Source/Modules/php.cxx

```diff
@@ -1050,11 +1429,13 @@ class PHP : public Language {
     DelWrapper(f);
     f = NULL;
 
+    if (wrapperType == membervar && Strstr(wname,"_get") != NULL)
```

Comment:
This checks for `_get` anywhere in the name, not just at the end.  Better to re-use the existing check (search `php.cxx` for `"_get"`).